### ISSUE TITLE
loss function in training phase

### ISSUE THREAD

**Issue by lisc199** [google/sky-optimization#8]
According to paper, I think that the groundtruth and output of model are continuous rather than binary. Is that right?

If it is right, which loss function was chosen to train the model? The CrossEntropy loss requires that the target be binary. 

Or is a soft label for CE loss?

**Comment by orlyliba** [comment]
Yes, we use continuous labels and outputs.  
We use a sigmoid cross entropy loss: https://www.tensorflow.org/api_docs/python/tf/nn/sigmoid_cross_entropy_with_logits#args

**Comment by misaka-10032** [comment]
> The CrossEntropy loss requires that the target be binary.

This is not true. Please read the argument.
https://www.tensorflow.org/api_docs/python/tf/nn/softmax_cross_entropy_with_logits#args_1
The link Orly gives has detailed math in it.

In the two class setting (sky / non-sky), the sigmoid cross entropy is the same as the softmax cross entroy. It's just a matter how you want to prepare your logits (in one channel vs in two channels).

**Comment by lisc199** [comment]
Got it. Thanks for your explaination.
